internal/ignition/ignition.go

```diff
@@ -591,6 +597,41 @@ func (g *installerGenerator) Generate(ctx context.Context, installConfig []byte,
 	return nil
 }
 
+func (g *installerGenerator) importClusterTLSCerts() error {
+	tlsDir := os.Getenv("ASSISTED_SERVICE_CLUSTER_TLS_CERTS_DIR")
+	if tlsDir == "" {
+		return nil
+	}
+	entries, err := os.ReadDir(tlsDir)
+	if err != nil && !os.IsNotExist(err) {
+		return err
+	}
+	if err := os.Mkdir(filepath.Join(g.workDir, "tls"), 0755); err != nil {
+		return err
+	}
+	tlsFS := os.DirFS(tlsDir)
+
+	for _, e := range entries {
+		if !e.Type().IsRegular() {
+			continue
+		}
+		f, err := tlsFS.Open(e.Name())
+		if err != nil {
+			return err
+		}
+		defer f.Close()
+		c, err := io.ReadAll(f)
+		if err != nil {
+			return err
+		}
+		err = os.WriteFile(filepath.Join(g.workDir, "tls", e.Name()), c, 0600)
```

Comment:
i know that os.WriteFile is actually the same as ioutil.WriteFile but in matter of consistency please move to ioutil.WriteFile